pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/ExistingRecordPatch.java

```diff
@@ -209,16 +208,6 @@ Object handleSingleNameValue() {
         if (Patch.patch(patch.getId(), patch.getUser(), patch.getNameValue(), dbElement)) {
             nameValues.addAll(setWorkflowRelatedFields(patch));
             exportToESWithId(patch.getId(), patch.getNameValue());
-            if (patch.getNameValue().getName().equals(DBConstants.OD_TISSUE_RECEIVED)) {
```

Comment:
Some PR comments here about why this is being deleted would be helpful so that reviewers understand where the code has moved to (or why it's no longer needed).

Reply:
was deleted because we use the sample accessioning now and not the tissue_received date anymore. change from cmi